security/advancedtls/advancedtls.go

```diff
@@ -35,10 +35,10 @@ import (
 	credinternal "google.golang.org/grpc/internal/credentials"
 )
 
-// VerificationFuncParams contains parameters available to users when
-// implementing CustomVerificationFunc.
+// HandshakeVerificationInfo contains parameters available to users when
+// implementing PostHandshakeVerificationFunc.
 // The fields in this struct are read-only.
-type VerificationFuncParams struct {
+type HandshakeVerificationInfo struct {
```

Comment:
I'm a little bit confused by naming - there are entries of 'xInfo' and 'xOptions'. It looks like that 'xInfo' is more about 'general purpose container' and 'xOptions' is more about 'container for func params' but I'd suggest double checking with Doug.

Reply:
I disagree with `xOptions` for this case - `xOptions` is for users to create configuration in our APIs.
This struct represents the input to a function that users can override - it's not `Options` or anything that is configured. Rather, the struct contains and represents information about the handshake verification steps that have already been taken, such as the raw certs, the built chains, the server name

Reply:
I think `Info` here is the best in this case